src/main/java/org/symphonyoss/symphony/messageml/MessageMLParser.java

```diff
@@ -58,6 +60,8 @@
 
   private int index;
 
+  private Set<String> elementIds = new HashSet<>();
```

Comment:
This shouldn't be initialized here. It should be instantiated on the .parse method instead